File: go/cafe/cafe.go
package cafe

// Cafe represents the metadata of a cafe
type Cafe struct {
	Name     string
	Drinks   []Drink
	CafeFood []Food
}

// Drink represents a drink you can get at the cafe
type Drink struct {
	Name  string
	Price int
}

func (d Drink) GetPrice() int {
	return d.Price
}

// Food represent a food item you can get at the cafe
type Food struct {
	Name     string
	Price    int
	Calories int
}

func (f Food) GetPrice() int {
	return f.Price
}

func (f Food) GetCalories() int {
	return f.Calories
}

// Init initializes our sample cafe
func (c *Cafe) Init() {
	c.Name = "Happy Coffee"
	c.Drinks = []Drink{
		{"Americano", 3},
		{"Cappuccino", 5},
		{"Latte", 5},
	}
	c.CafeFood = []Food{
		{"chocolate chip cookies", 5, 78},
		{"blueberry muffin", 5, 467},
		{"fruit cup", 7, 124},
	}
}

File: python/cafe/cafe.py
class Cafe:
	def __init__(self):
		self.Name = "Happy Coffee"
		self.Drinks = [
			Drink("Americano", 3),
			Drink("Cappuccino", 5),
			Drink("Latte", 5)
		]
		self.CafeFood = [
			Food("chocolate chip cookies", 5, 78),
			Food("blueberry muffin", 5, 467),
			Food("fruit cup", 7, 124),
		]


class Drink:
	def __init__(self, name: str, price: int):
		self.Name = name 
		self.Price = price

	def getName(self) -> str:
		return self.Name

	def getPrice(self) -> int:
		return self.Price

	
class Food:
	def __init__(self, name: str, price: int, calories: int):
		self.Name = name
		self.Price = price
		self.Calories = calories

	def getName(self) -> str:
		return self.Name

	def getPrice(self) -> int:
		return self.Price

	def getCalories(self) -> int:
		return self.Calories
